pkg/orderedmap/convert.go

```diff
@@ -74,7 +74,13 @@ func (c Conversion) fromUnorderedMaps(object interface{}) interface{} {
 			typedObj[i] = c.fromUnorderedMaps(item)
 		}
 		return typedObj
-
+	case []map[string]interface{}:
```

Comment:
```suggestion

	// some sources (e.g. toml library) yield specific type of slices.
	// slices in Go are not covariant, so each flavor of slice must have its own case, here.
	// process these exactly the same way we do for generic slices (prior case)
	case []map[string]interface{}:
```

For now: Leave ourselves a note about why we have two cases with the same body.
Soon: we'll eventually upgrade to 1.18. At that point, we might be able to make this collection generic. 😁